Authentication/logout.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Arcade Store</title>
+    <script src="https://kit.fontawesome.com/4a9487b343.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://nifty-tesla-85dda2.netlify.app/Components/components.css">
+    <link rel="stylesheet" href="../common.css">
+    <link rel="stylesheet" href="auth.css">
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class="nav-logo">
+            <img src="../Assets/nav-logo.png" alt="logo" />
+        </div>
+        <div class="nav-items">
+            <a class="nav-link sm-title hide-md" href="../Products/products.html">Shop</a>
+            <a class="nav-link sm-title hide-md" href="../index.html">Categories</a>
+        </div>
+        <div class="nav-space"></div>
+        <div class="nav-icon-items">
+            <div class="nav-search-bar hide-sm">
+                <input class="search-bar" type="text" placeholder="Search..">
+                <i class="fa fa-brands fa-searchengin"></i>
+            </div>
+            <a href="login.html">
+                <button id="dark-bg-icon" class="btn icon-btn hide-md">
+                    <i class="fa fa-solid fa-user"></i>
+                </button>
+            </a>
+            <a href="../Wishlist/wishlist.html">
+                <button id="dark-bg-icon" class="btn icon-btn hide-md">
+                    <i class="fa fa-solid fa-heart"></i>
+                </button>
+            </a>
+            <a href="../Cart/cart-page.html">
+                <button id="dark-bg-icon" class="btn icon-btn hide-md">
+                    <i class="fas fa-shopping-cart"></i>
+                </button>
+            </a>
+            <button id="dark-bg-icon" class="btn icon-btn show-md">
```

Comment:
Change: id's are unique multiple elements shouldn't have the same id
Also if you are using classes for styling keep it consistent and don't use id for styling then